### DISCUSSION TITLE
mirros for packages.sury.org

### DISCUSSION THREAD

**Discussion by Cyrille37** [oerdnj/deb.sury.org#2005] [General]
Hello

@oerdnj Thousands of thanks for your service 🙂 

Community users : Do you know if some packages.sury.org public MIRRORs exist ? The service must be overloaded if not 😞  (I've read about the [use of Cloudflare (or Bunny)](https://github.com/oerdnj/deb.sury.org/wiki/Frequently-Asked-Questions#repository-mirroring)).
Many hosting providers deploy some mirrors but I haven't found any that mirroring packages.sury.org ...

Any idea ? Am I wrong ?

Thanks & cheers

**Comment by oerdnj** [comment]
Yeah, the backend is hosted by Mythic Beasts and the front end is cached by BunnyNet. Does the attached image for Debian packages answer your question?

Sadly, I don't have data from the PPAs, that's handled by Canonical.

![image](https://github.com/oerdnj/deb.sury.org/assets/130731/664f5fb5-c138-499c-8d7c-2dfa89023fa8)

**Comment by bashgeek** [comment]
To answer your question, not really necessary to have a public mirror, Bunny should be able to and can handle all of it with ease, especially with that quite good cache rate.

@oerdnj If you don't mind me asking, are you paying for that or is Bunny providing this?

**Reply by oerdnj** [reply]
I got a generous credit from BunnyCDN and 100% discount from Mythic Beasts.
